src/test/java/org/jboss/arquillian/extension/jacoco/test/unit/FilterComposerTestCase.java

```diff
@@ -68,36 +68,36 @@ public void should_exclude_asset_not_matching_include_pattern() throws Exception
    }
 
    @Test
-   public void should_include_asset_not_matching_exclude_pattern() throws Exception
+   public void should_include_asset_matching_one_of_include_patterns() throws Exception
    {
       // given
-      final Filter<ArchivePath> filter = createComposedFilter(null, "org.arquillian.extension.jacoco.client.*");
+      final Filter<ArchivePath> filter = createComposedFilter("org.arquillian.extension.jacoco.client.*, org.arquillian.extension.integration.*", null);
 
       // when
-      final boolean include = filter.include(ArchivePaths.create("org/arquillian/extension/jacoco/container/CoverageDataCommand.class"));
+      final boolean include = filter.include(ArchivePaths.create("org/arquillian/extension/jacoco/client/ManifestAsset.class"));
```

Comment:
By refactoring friendly I mean that we changed package of this class - which is some kind of refactoring. Now, since we rely on string representation for the path it does not reflect reality anymore as string path is wrong. We did that already for the higher level test. Please have a look at [adjusted test](https://github.com/arquillian/arquillian-extension-jacoco/pull/31/commits/fb488f328d7099c82a47b66ec51542fbd9356f58#diff-3000a9437ea469d92e6b9063dc369723R41) and make similar changes in all the other tests.